pkg/jx/cmd/install.go

```diff
@@ -515,6 +515,20 @@ func (options *InstallOptions) Run() error {
 		return errors.Wrap(err, "configuring minikube from kubectl context")
 	}
 
+	if options.Flags.DockerRegistryOrg == "" {
```

Comment:
@carlossg I would move this to `checkFlags` method. We try to keep the default flags configuration in a central place. Thanks